### PR TITLE
141 reg filtering

### LINKED ISSUE #141: set up student reg table with filters like schooladmin

### PR DESCRIPTION
**Pull request by oceancypress**
added filtering to registration
checkout registration

Closes #141

### CODE REVIEW

File: student-info-system/student/templates/student/registration.html [hunk 1]
```diff
@@ -4,19 +4,32 @@
...
+      </form>
+    {% endif %}
+
+  <form action="{% url 'student:registration' %}" method="post">
+    {% csrf_token %}
+    <select name="semester">
+    {% for sem in semesters %}
+      <option value="{{sem.id}}" {% if semester == sem.id %}SELECTED{% endif %}>{{ sem.name }}</option>
+    {% endfor %}
+    </select>
```
Comment (bjmckenz) [inline student-info-system/student/templates/student/registration.html:31]: I strongly suggest you look at filtered_view in sa/views, and how it interacts with the table, filter, and HTML. It supports multiple tables per page (mostly) and is kinda consistent in its use.

There are two use cases -- one for (say) students() which is a single table, and (say) student() which is some data and 1+ tables.

**Review by bjmckenz — CHANGES_REQUESTED** [review]
please don't delete the .yml. 

Up to you on filtered_table, but I cna't see a reason not to do it.

File: student-info-system/student/templates/student/registration.html (hunk 1 above)
Comment (oceancypress) [reply student-info-system/student/templates/student/registration.html:31]: I need more info, I cant find a filtered view to try to figure out the context.

**Review by bjmckenz — APPROVED** [review]
great

**Comment by oceancypress** [comment]
Going to wait to push this until the filtered_table is moved to sis so I can merge and utilize

**Pull request merged**